rdkafka.cmake

```diff
@@ -0,0 +1,51 @@
+# Build only the static C library
+# Does not build C++ library
+# Does not build shared libraries
+# Does not run tests
+# Does not generate documentation
+
+
+# Protect multiple-inclusion of this CMake file
+if( NOT TARGET rdkafka::rdkafka )
+
+# librdkafka depends on pthread and zlib
+find_package(Threads REQUIRED)
+find_package(ZLIB    REQUIRED)  # sudo apt install zlib1g-dev
+
+# The librdkafka build is based on tools './configure' and 'make'
+include(ExternalProject)
```

Comment:
please rebase from master for latest cmakeLists.txt changes

Reply:
done

Reply:
done